gateway-ha/src/test/java/io/trino/gateway/ha/router/TestRoutingGroupSelector.java

```diff
@@ -127,9 +125,17 @@ public void testByRoutingRulesEngineFileChange()
                         + "actions:\n"
                         + "  - \"result.put(\\\"routingGroup\\\", \\\"etl\\\")\"");
         fw.close();
-        file.getPath();
+        boolean successfullySetLastModifiedTime = file.setLastModified(0);
```

Comment:
Setting 0 doesn't make sense. Please the last modified time + 1. 
Add `assertThat(file.setLastModified(0)).isTrue()`